.github/PULL_REQUEST_TEMPLATE/security_related_pull_request_template.md

```diff
@@ -0,0 +1,60 @@
+---
+name: "Security-related Pull Request"
+about: "This pull request template is intended to be used when there is a change in our attack surface."
+title: "Security-related Pull Request"
+labels: "security"
+assignees: "ronaldheft-usds", "td-usds", "jimduff-usds", "RickHawesUSDS"
+---
+
+
+This PR contains changes which require special attention to security-related aspects
+
+
+## Changes
+-
+-
+
+## Specific Security-related subjects a reviewer should pay specific attention to
+
+- Newly introduced enpoints
```

Comment:
Why not just add this to security review to our standard PRs, so it happens every time?

You might want to rearrange the questions a bit:   Put the endpoints ,auth, infrastructure, dependencies questions first. Then just have a sentence that says, "If you answered yes to any of those questions you must conduct a security review, as follows:   ... then put the question about threats and mitigations.

Reply:
Good idea! See 6f3f3ffe

Reply:
I like the idea of making this part of the normal PR process and not have a separate template which now people can _not_ select in error; see 7770317d9008294171e13530415c6af5abe94379